src/main/java/core/basesyntax/classes/functional/Colors.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.classes.functional;
+

```

Comment:
Remove this unnecessary empty line to maintain a clean and organized code structure.